AWSS3/AWSS3Model.h

```diff
@@ -6295,14 +6956,14 @@ typedef NS_ENUM(NSInteger, AWSS3Types) {
 @property (nonatomic, strong) NSString * _Nullable identifier;
 
 /**
- <p>The Amazon Resource Name (ARN) of the Amazon SQS queue to which Amazon S3 will publish a message when it detects events of the specified type.</p>
+ <p>The Amazon Resource Name (ARN) of the Amazon SQS queue to which Amazon S3 publishes a message when it detects events of the specified type.</p>
  */
 @property (nonatomic, strong) NSString * _Nullable queueArn;
 
 @end
 
 /**
- 
+ <p>This data type is deprecated. Use <a href="https://docs.aws.amazon.com/AmazonS3/latest/API/API_QueueConfiguration.html">QueueConfiguration</a> for the same purposes. This data type specifies the configuration for publishing messages to an Amazon Simple Queue Service (Amazon SQS) queue when Amazon S3 detects specified events. </p>
```

Comment:
Do we add deprecated warning in codegened models? I donot think we have, just checking..

Reply:
That came from the model itself, so we just pass it through.